lighthouse-treemap/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../tsconfig",
+  "compilerOptions": {
+    "typeRoots": [
```

Comment:
I don't think this actually includes the types like you want it to, does it? That might be your problem, most of our types aren't [*packages*](https://www.typescriptlang.org/tsconfig#typeRoots), they're just files that should be in `include`, no?

Reply:
awesome, thanks!